source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -391,6 +460,20 @@ def _direct_path(
         direct_path_type,
         request_id,
     ):
+        if self.use_direct_path_service_backup(
```

Comment:
handle the case where `direct_path_type == direct`

Reply:
:+1: